src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -63,62 +63,62 @@ components:
         issueDate:
           type: string
           format: date
-          description: The issue date of the Voter Authority Card
+          description: The issue date of the Voter Authority Certificate
           example: '2022-06-01'
         requestDateTime:
           type: string
           format: date-time
-          description: The date time the Voter Authority Certificate was requested
+          description: The date time the Voter Authority Certificate print request was requested
           example: '2022-06-01T12:23:03.000Z'
-        cardFirstname:
+        firstName:
           type: string
           maxLength: 255
           description: First name of the Elector
           example: John
-        cardMiddlenames:
+        middleNames:
           type: string
           maxLength: 255
           description: Middle names of the Elector
           example: Malcolm
-        cardSurname:
+        surname:
           type: string
           maxLength: 255
           description: Surname of the Elector
           example: Smith
         certificateLanguage:
           $ref: '#/components/schemas/CertificateLanguage'
-        deliveryOption:
```

Comment:
`deliveryOption`, `deliveryName` and `deliveryAddess` have been bundled up into a single property `delivery` of type `CertificateDelivery` (I'm open to suggestions of better names perhaps ?)